charts/registry/templates/registry/registry-ingress.yaml

```
@@ -19,17 +19,14 @@
 # SPDX-License-Identifier: Apache-2.0
 ###############################################################
 
-{{- $deployment_name := printf "cx-%s-registry" .Release.Name }}
-{{- $svc_name := printf "%s-svc" $deployment_name }}
-{{- $ingr_name := printf "%s-ingr" $deployment_name }}
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: {{ $ingr_name }}
+  name: {{ include "chart.fullname" . }}-ingr
```

The suffix is not necessary, as you just create one ingress with your helm chart ;-) 

```suggestion
  name: {{ include "chart.fullname" . }}
```

---

This could be also applied to all the other resources (except the keycloak stuff 😉)